catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/faceting/FacetedAttributeResult.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.operation.faceting;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FacetedAttributeResult {
```

Comment:
I'll add the Javadoc.
I had a discussion about this with Phil. Since we're not adding this to 'catalog-core-api` for the time being, none of it needs to be marked experimental. I figured we could mark the primary "interface-ish" classes as Experimental so no one goes off using on them without expecting change.